pypeit/ech_coadd.py

```diff
@@ -0,0 +1,201 @@
+
+import numpy as np
+from astropy.io import fits
+from astropy import units
+import matplotlib.pyplot as plt
+
+from pypeit.core import coadd
+from pypeit.core import load
+from pypeit import msgs
+
+from linetools.spectra.utils import collate
+from linetools.spectra.xspectrum1d import XSpectrum1D
+
+## ToDo: change it to a CLASS and modify coadd_1dspec.py
+def load_spec_order(fname,objid=None,order=None,extract='OPT',flux=True):
```

Comment:
This should probably be in core/load.py, right?

Reply:
done